### PR TITLE
Mails styling reset password

### PR DESCRIPTION
**Pull request by adititipnis**
image

### CODE REVIEW

**Review by JijoBose — CHANGES_REQUESTED** [review]
Lock icon missing from the mailer,

Screenshot 2024-01-19 at 2 12 16 PM

**Review by JijoBose — DISMISSED** [review]
Screenshot 2024-01-19 at 2 19 12 PM

File: app/assets/stylesheets/mailers.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+}
+.container {
+  text-align: center;
+  width: 640px;
+    font-family: 'Manrope';
+    font-style: normal;
+    font-weight: 400;
+    font-size: 14px;
+    line-height: 19px;
+    text-align: center;
```
Comment (nisusam) [inline app/assets/stylesheets/mailers.css:37]: Duplicate.
Fix Indentation.

File: app/javascript/images/Banner Animation.svg
Comment (nisusam) [inline app/javascript/images/Banner Animation.svg:1]: Remove this file if unused

File: app/javascript/images/Miru Logo With Text.svg
Comment (nisusam) [inline app/javascript/images/Miru Logo With Text.svg:1]: Remove this file if unused. Also please use filenames without spaces.

**Review by nisusam — CHANGES_REQUESTED** [review]

**Comment by apoorv1316** [comment]
@adititipnis Please work on review comments

File: app/assets/stylesheets/mailers.css [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+}
+.container {
+  text-align: center;
+  width: 640px;
+  font-family: 'Manrope';
+  font-style: normal;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+  text-align: center;
```
Comment (nisusam) [inline app/assets/stylesheets/mailers.css:37]: This is duplicate.

File: app/assets/stylesheets/mailers.css [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      width: 129px;
+      height: 24px;
+      font-family: system-ui, -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+      font-weight: 700;
+      font-size: 12px;
+      line-height: 16px;
+      color: #FFFFFF;
+      text-decoration: none;
+      display: inline-block;
+      display: flex;
```
Comment (nisusam) [inline app/assets/stylesheets/mailers.css:56]: `display` property used couple of times

**Review by nisusam — APPROVED** [review]
Fix the comments. Rest LGTM!

**Pull request merged**